clang/lib/Sema/SemaExpr.cpp

```diff
@@ -3858,7 +3858,8 @@ static Expr *BuildFloatingLiteral(Sema &S, NumericLiteralParser &Literal,
   using llvm::APFloat;
   APFloat Val(Format);
 
-  APFloat::opStatus result = Literal.GetFloatValue(Val);
+  APFloat::opStatus result =
```

Comment:
Does this work right in templates?  Can we get a test that validates that a NTTP or Dependent value where the rounding mode is set either around or inside the function does the math right?